pages/glossary/disabled.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Disabled
+key: disabled
+---
+
+An element that is normally [perceivable][] and [operable][], but has been rendered [inoperable][operable] either temporarily or permanently, is considered disabled. While there exists several techniques through which elements can be disabled, the most typical are:
```

Comment:
@kasperisager This definition is ambiguous, as it leaves room for other unspecified ways of disabling elements. That's not allowed in the applicability of a rule. I suggest we keep it simpler and stick to the two items you have here, leave out the part about perceivable / (in)operable. If you want to leave room for a future "disabled" definition that does include those, we can call this definition "programatically disabled" or something along those lines. Not my preference, I'd rather not try to anticipate the future, but I wouldn't be opposed to it either.

Reply:
I'd be fine with sticking to `disabled` and `aria-disabled` 👍

Reply:
@WilcoFiers Let me know if 174c64e583fc05dcf3d75d2260ec3a80fdc0957d does the trick. I've left in "perceivable" and "operable" but otherwise restricted the ways in which elements can be "disabled".